platform/entanglement/src/services/CopyTask.js

```diff
@@ -185,6 +203,10 @@ define(
         CopyTask.prototype.perform = function(){
             this.deferred = this.$q.defer();
 
+            if (!this.parent.hasCapability('composition')){
```

Comment:
Actually that's a good catch. I moved this from elsewhere in the code, without checking whether it was really necessary, but it is essentially duplicating a check in CopyService.validate. Furthermore, it's not even correct, should be using the policy and not the capability. I'll remove it.